projects/plugins/jetpack/tests/php/test-get-modules.php

```diff
@@ -1,12 +1,12 @@
 <?php
 /**
- * Test get_modules methods in Jetpack class
+ * Test module related methods in Jetpack and Jetpack_Admin class.
```

Comment:
Not sure if this is the best place to add my unit tests, but given (by intuition) me searching for module related tests would first look here it seemed to fit.
@leogermani Totally ok to change it if you prefer to leave this class as is.